### ISSUE TITLE
[BUG] no server running on /tmp/tmux-997/default - Valheim

### ISSUE THREAD

**Issue by marcleblanc2** [GameServerManagers/LinuxGSM#3397]
```
$ ./vhserver st
[  OK  ] Updating vhserver: Checking for update: SteamCMD

No update available
* Local build: 6315977
* Remote build: 6315977
* Branch: public
https://steamdb.info/app/896660/

[ .... ] Starting vhserver: marctestno server running on /tmp/tmux-997/default
[ FAIL ] Starting vhserver: Unable to start marctest
```

## User Story

As a LinuxGSM user and Valheim server admin, I want LinuxGSM to be able to successfully start the Valheim server.

## Basic info

* **Distro:** [Ubuntu, 20.04 LTS, amd64, focal image build on 2021-03-15]
* **Game:** [Valheim]
* **Command:** [command_start.sh]
* **LinuxGSM version:** [v21.2.2]
* ldd (Ubuntu GLIBC 2.31-0ubuntu9.2) 2.31
* tmux 3.0a
* AWS EC2, t3.medium, 2 vCPUs, 4 GiB RAM, gp3 SSD

## Further Information

I see the back and forth of adding and removing the double quotes on this line, to fix one game but break another. With the double quotes, I wasn't able to start the Valheim server, so I removed them, and it works.

[command_start.sh#L75](https://github.com/GameServerManagers/LinuxGSM/blob/532443eafeee33706399a980070be70e2d5693fd/lgsm/functions/command_start.sh#L75)
`tmux new-session -d -x ${sessionwidth} -y ${sessionheight} -s ${sessionname} ${preexecutable} ${executable} ${startparameters} 2> "${lgsmlogdir}/.${selfname}-tmux-error.tmp"`

This seems related to:
- #3239 
- #3375 

## To Reproduce

Steps to reproduce the behaviour:
1. `$ ./vhserver st`

## Expected behaviour
Game server starts

**Comment by deathstar** [comment]
I tried to use the update command as well and the server is not updating. Since the client auto-updates through steam, I am now getting "incompatible version" and valheim is no longer playable :(. Please let me know how I can update my valheim server. Thank you!

**Comment by deathstar** [comment]
Figured it out. Since I had modified and used serverfiles/start_server.sh to start the server, this instance did not auto-update like i had enabled. The proper way to manage the valheim server is by using ./vhserver start, after using the template from ~/lgsm/config-default/config-lgsm/vhserver/_default.cfg, i was able to create my config in ~/lgsm/config-lgsm/vhserver/vhserver.cfg which will be referenced when using ./vhserver start.

**Comment by dvankley** [comment]
@marcleblanc2 Your fix also gets the server running for me, but it appears to mess up the command parsing so that quoted options passed to `valheim_server.x86_64` get mangled, i.e. server name and save file location.

**Comment by dvankley** [comment]
Now I put the quotes back and it works? Guess I need to investigate more...

**Comment by CoderDanS** [comment]
can confirm your fix works for me as well @marcleblanc2. Strange that this is an issue.

**Comment by purplg** [comment]
I also had this issue and the fix @marcleblanc2 posted resolved the issue, the real culprit for me was having a single quote `'` in my server name. The arguments passed to the server get passed as single quotes, for example `./valheim_server.x86_64 -name 'Someone's Server'`.

My solution was just removed the `'` in the title.

**Comment by Claiyc** [comment]
Does not seem to be an issue (anymore). Probably fixed by #3370
